rosdep/base.yaml

```diff
@@ -2806,6 +2806,14 @@ liblapack-dev:
 liblapacke-dev:
   debian: [liblapacke-dev]
   ubuntu: [liblapacke-dev]
+liblcm-dev:
+  debian: [liblcm-dev]
+  fedora: [liblcm-devel]
+  ubuntu: [liblcm-dev]
```

Comment:
Since we still support Kinetic on Xenial, and it looks like these packages aren't available there, we need to change this to:

```
ubuntu:
  '*': [liblcm-dev]
  xenial: null
```

The same goes for the key below.